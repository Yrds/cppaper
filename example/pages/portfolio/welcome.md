# my first portfolio

| table | table |
| ----- | ----- |
| test  | test  |
