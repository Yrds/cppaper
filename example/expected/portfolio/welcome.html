<h1>my first portfolio</h1>
